src/main/java/tests/ShopServiceTest.java

```diff
@@ -0,0 +1,110 @@
+package tests;
+
+import core.basesyntax.*;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class ShopServiceTest {
```

Comment:
The test class name should reflect the class being tested. Consider renaming the class to `ShopServiceImplTest` to clearly indicate that this class is intended to test the `ShopServiceImpl`.